SygicUIKit/Classes/Views/SpeedSigns/SYUICurrentSpeedView.swift

```diff
@@ -30,6 +30,13 @@ public class SYUICurrentSpeedView: UIView {
     public var currentSpeed = 0 {
         didSet {
             currentSpeedLabel.attributedText = NSAttributedString(string: String(currentSpeed), attributes: [NSAttributedString.Key.font: UIFont(name: "DINCondensed-Bold", size: 28)!, NSAttributedString.Key.baselineOffset: -3])
+            updateProgress()
+        }
+    }
+    
+    public var speedLimit = 90 {
```

Comment:
Dal by som sem aspon nejaky koment co znamena tento speed limit. Tiez ma to trochu zmiatlo ze preco je v current speed view nejaky speed limit. Ze to sluzi na ten progress bar.